<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment> * OpenRPT report writer and rendering engine
 * Copyright (C) 2001-2018 by OpenMFG, LLC
 *
 * This library is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public
 * License as published by the Free Software Foundation; either
 * version 2.1 of the License, or (at your option) any later version.
 *
 * This library is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
 * Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public
 * License along with this library; if not, write to the Free Software
 * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 * Please contact info@openmfg.com with any questions on this license.</comment>
 <class>EditPreferences</class>
 <widget class="QDialog" name="EditPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QHBoxLayout">
   <item>
    <widget class="QTabWidget" name="_tab">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>100</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="_reportTab">
      <attribute name="title">
       <string>Report</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout">
         <item>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Language :</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>_cbLanguage</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="_cbLanguage"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="_bgDefaultFont">
           <property name="title">
            <string>Default Font</string>
           </property>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>11</number>
            </property>
            <property name="topMargin">
             <number>11</number>
            </property>
            <property name="rightMargin">
             <number>11</number>
            </property>
            <property name="bottomMargin">
             <number>11</number>
            </property>
            <item>
             <widget class="QLineEdit" name="_leDefaultFont">
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="_btnChangeFont">
              <property name="text">
               <string>&amp;Font...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="_gbGridOptions">
           <property name="title">
            <string>Grid Options</string>
           </property>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>11</number>
            </property>
            <property name="topMargin">
             <number>11</number>
            </property>
            <property name="rightMargin">
             <number>11</number>
            </property>
            <property name="bottomMargin">
             <number>11</number>
            </property>
            <item>
             <widget class="QCheckBox" name="_cbShowGrid">
              <property name="text">
               <string>Show grid</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="_cbSnapGrid">
              <property name="text">
               <string>Snap to grid</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="_gbGridSize">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Grid Size Interval</string>
              </property>
              <layout class="QGridLayout">
               <property name="leftMargin">
                <number>11</number>
               </property>
               <property name="topMargin">
                <number>11</number>
               </property>
               <property name="rightMargin">
                <number>11</number>
               </property>
               <property name="bottomMargin">
                <number>11</number>
               </property>
               <property name="spacing">
                <number>6</number>
               </property>
               <item row="2" column="0">
                <spacer>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::MinimumExpanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="2" column="1">
                <widget class="QCheckBox" name="_cbGridSymetrical">
                 <property name="text">
                  <string>Symetrical values</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="_lblGridSizeY">
                 <property name="text">
                  <string>Y Interval:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="buddy">
                  <cstring>_leGridSizeY</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="_leGridSizeX"/>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="_leGridSizeY"/>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="_lblGridSizeX">
                 <property name="text">
                  <string>X Interval:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="buddy">
                  <cstring>_leGridSizeX</cstring>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_systemTab">
      <attribute name="title">
       <string>System</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="_gbPageSetup">
           <property name="title">
            <string>Page Setup</string>
           </property>
           <widget class="QWidget" name="horizontalLayoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>161</width>
              <height>68</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="1" column="1">
              <widget class="QRadioButton" name="_rbPortrait">
               <property name="text">
                <string>Portrait</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="_cbPageSize"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="_lbSize">
               <property name="text">
                <string>Page Size</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="_lbOrientation">
               <property name="text">
                <string>Orientation</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QRadioButton" name="_rbLandscape">
               <property name="text">
                <string>Landscape</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="_btnOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_btnCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>_btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>EditPreferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_btnOk</sender>
   <signal>clicked()</signal>
   <receiver>EditPreferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_cbGridSymetrical</sender>
   <signal>toggled(bool)</signal>
   <receiver>_lblGridSizeY</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_cbGridSymetrical</sender>
   <signal>toggled(bool)</signal>
   <receiver>_leGridSizeY</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
